apps/projects/app/components/Panel/ReviewWork/ReviewWork.js

```diff
@@ -124,17 +123,22 @@ const ReviewWork = ({ issue }) => {
         <React.Fragment>
           <FieldTitle>Submission Status</FieldTitle>
           <FieldText>
-            {work.review.accepted ? (
-              <div css="display: flex; align-items: center">
-                <IconCheck color={`${theme.positive}`} css="margin-top: -4px; margin-right: 8px"/>
-                <Text color={`${theme.positive}`}>Accepted</Text>
-              </div>
-            ) : (
-              <div css="display: flex; align-items: center">
-                <IconCross color={`${theme.negative}`} css="margin-top: -4px; margin-right: 8px" />
-                <Text color={`${theme.negative}`}>Rejected</Text>
-              </div>
-            )}
+            <div css="display: flex; align-items: center">
+              {work.review.accepted ? (
+                <>
+                  <IconCheck color={`${theme.positive}`} css="margin-top: -4px; margin-right: 8px"/>
+                  <Text color={`${theme.positive}`}>Accepted</Text>
+                </>
+              ) : (
+                <>
+                  <IconCross color={`${theme.negative}`} css="margin-top: -4px; margin-right: 8px" />
+                  <Text color={`${theme.negative}`}>Rejected</Text>
+                </>
+              )}
+              <Text color={`${theme.contentSecondary}`} css={`margin-left: ${GU}px`}>
+                {formatDate(new Date(work.review.reviewDate), 'd MMM yy, h:MM a (z)')}
+              </Text>
+            </div>
```

Comment:
We could put this new `Status` component in `components/Panel` or maybe in `components/Shared` and then use it in both of these spots.

Reply:
Done in PR #1698